packages/web/src/components/search/CategorySearch.js

```diff
@@ -919,8 +941,18 @@ class CategorySearch extends Component {
 
 	render() {
 		const { currentValue } = this.state;
-		const { theme, themePreset, size } = this.props;
+		const {
+			theme,
+			themePreset,
+			size,
+			enableQuerySuggestions,
+			querySuggestions,
+			showDistinctSuggestions,
+		} = this.props;
 		const finalSuggestionsList = this.parsedSuggestions;
+		const topSuggestions = enableQuerySuggestions
+			? getTopSuggestions(querySuggestions, currentValue, showDistinctSuggestions)
```

Comment:
@ShahAnuj2610 You're computing it two times, please avoid code repetition.
Use a getter method to get query suggestions.